Could not find keychain_item in security_framework_sys

```
$ sw_vers
ProductName:    Mac OS X
ProductVersion: 10.15.7
BuildVersion:   19H2

$ rustc --version
rustc 1.49.0 (e1884a8e3 2020-12-29)

$ cargo --version
cargo 1.49.0 (d00d64df9 2020-12-05)

$ git clone https://github.com/kornelski/rust-security-framework
$ cd rust-security-framework
$ cargo build --target x86_64-apple-ios

...

Compiling security-framework v2.1.0 (/Users/jenkins/sergeivii-test-a/rust-security-framework/security-framework)
error[E0432]: unresolved import `security_framework_sys::keychain_item`
  --> security-framework/src/item.rs:14:29
   |
14 | use security_framework_sys::keychain_item::SecItemCopyMatching;
   |                             ^^^^^^^^^^^^^ could not find `keychain_item` in `security_framework_sys`
```

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -29,6 +29,33 @@ jobs:
         uses: actions-rs/cargo@v1
         with:
           command: test
+          args: --all-features
+
+      - name: Check default features
+        uses: actions-rs/cargo@v1
+        with:
+          command: check
+
+  ios:
+    name: iOS compile-check
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v2
+
+      - name: Install stable toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+          target: aarch64-apple-ios
+          override: true
+
+      - name: Run check
+        uses: actions-rs/cargo@v1
+        with:
+          command: check
+          args: --all-features -p security-framework --target aarch64-apple-ios
 
   lints:
     name: Lints
@@ -49,3 +76,4 @@ jobs:
         uses: actions-rs/cargo@v1
         with:
           command: clippy
+          args: --all-features

--- a/security-framework-sys/Cargo.toml
+++ b/security-framework-sys/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "security-framework-sys"
-version = "2.1.0"
+version = "2.1.1"
 authors = ["Steven Fackler <sfackler@gmail.com>", "Kornel <kornel@geekhood.net>"]
 license = "MIT OR Apache-2.0"
 description = "Apple `Security.framework` low-level FFI bindings"

--- a/security-framework-sys/src/base.rs
+++ b/security-framework-sys/src/base.rs
@@ -1,4 +1,5 @@
 use core_foundation_sys::base::OSStatus;
+#[cfg(target_os = "macos")]
 use core_foundation_sys::string::CFStringRef;
 use std::os::raw::c_void;
 

--- a/security-framework-sys/src/certificate.rs
+++ b/security-framework-sys/src/certificate.rs
@@ -1,11 +1,16 @@
+#[cfg(target_os = "macos")]
 use core_foundation_sys::array::CFArrayRef;
 use core_foundation_sys::base::{CFAllocatorRef, CFTypeID, OSStatus};
 use core_foundation_sys::data::CFDataRef;
+#[cfg(target_os = "macos")]
 use core_foundation_sys::dictionary::CFDictionaryRef;
+#[cfg(target_os = "macos")]
 use core_foundation_sys::error::CFErrorRef;
 use core_foundation_sys::string::CFStringRef;
 
-use crate::base::{SecCertificateRef, SecKeyRef};
+use crate::base::SecCertificateRef;
+#[cfg(target_os = "macos")]
+use crate::base::SecKeyRef;
 
 extern "C" {
     #[cfg(target_os = "macos")]

--- a/security-framework-sys/src/identity.rs
+++ b/security-framework-sys/src/identity.rs
@@ -1,4 +1,6 @@
-use core_foundation_sys::base::{CFTypeID, CFTypeRef, OSStatus};
+#[cfg(target_os = "macos")]
+use core_foundation_sys::base::CFTypeRef;
+use core_foundation_sys::base::{CFTypeID, OSStatus};
 
 use crate::base::{SecCertificateRef, SecIdentityRef, SecKeyRef};
 

--- a/security-framework-sys/src/import_export.rs
+++ b/security-framework-sys/src/import_export.rs
@@ -1,10 +1,14 @@
 use core_foundation_sys::array::CFArrayRef;
-use core_foundation_sys::base::{CFTypeRef, OSStatus};
+#[cfg(target_os = "macos")]
+use core_foundation_sys::base::CFTypeRef;
+use core_foundation_sys::base::OSStatus;
 use core_foundation_sys::data::CFDataRef;
 use core_foundation_sys::dictionary::CFDictionaryRef;
 use core_foundation_sys::string::CFStringRef;
+#[cfg(target_os = "macos")]
 use std::os::raw::c_uint;
 
+#[cfg(target_os = "macos")]
 use crate::base::{SecAccessRef, SecKeychainRef};
 
 #[cfg(target_os = "macos")]

--- a/security-framework-sys/src/key.rs
+++ b/security-framework-sys/src/key.rs
@@ -41,15 +41,17 @@ macro_rules! names {
             $(pub static $x: SecKeyAlgorithm;)*
         }
 
-        #[non_exhaustive]
         pub enum Algorithm {
-            $( $i ),*
+            $( $i, )*
+            #[doc(hidden)]
+            _NonExhaustive,
         }
 
         impl From<Algorithm> for SecKeyAlgorithm {
             fn from(m: Algorithm) -> Self {
                 unsafe { match m {
                     $( Algorithm::$i => $x, )*
+                    Algorithm::_NonExhaustive => kSecKeyAlgorithmRSASignatureMessagePSSSHA512,
                 } }
             }
         }

--- a/security-framework-sys/src/keychain.rs
+++ b/security-framework-sys/src/keychain.rs
@@ -1,7 +1,11 @@
-use core_foundation_sys::base::{Boolean, CFTypeID, CFTypeRef, OSStatus};
+#[cfg(target_os = "macos")]
+use core_foundation_sys::base::CFTypeRef;
+use core_foundation_sys::base::{Boolean, CFTypeID, OSStatus};
 use std::os::raw::{c_char, c_uint, c_void};
 
-use crate::base::{SecAccessRef, SecKeychainItemRef, SecKeychainRef};
+#[cfg(target_os = "macos")]
+use crate::base::SecKeychainItemRef;
+use crate::base::{SecAccessRef, SecKeychainRef};
 
 pub const SEC_KEYCHAIN_SETTINGS_VERS1: c_uint = 1;
 

--- a/security-framework-sys/src/lib.rs
+++ b/security-framework-sys/src/lib.rs
@@ -1,6 +1,5 @@
 #![allow(bad_style)]
 
-
 #[cfg_attr(
     any(target_os = "macos", target_os = "ios"),
     link(name = "Security", kind = "framework")

--- a/security-framework-sys/src/lib.rs
+++ b/security-framework-sys/src/lib.rs
@@ -24,9 +24,7 @@ pub mod identity;
 pub mod import_export;
 pub mod item;
 pub mod key;
-#[cfg(target_os = "macos")]
 pub mod keychain;
-#[cfg(target_os = "macos")]
 pub mod keychain_item;
 pub mod policy;
 pub mod random;

--- a/security-framework-sys/src/secure_transport.rs
+++ b/security-framework-sys/src/secure_transport.rs
@@ -1,6 +1,8 @@
 use core_foundation_sys::array::CFArrayRef;
 use core_foundation_sys::base::CFAllocatorRef;
-use core_foundation_sys::base::{Boolean, CFTypeRef, OSStatus};
+#[cfg(target_os = "macos")]
+use core_foundation_sys::base::CFTypeRef;
+use core_foundation_sys::base::{Boolean, OSStatus};
 use std::os::raw::{c_char, c_int, c_void};
 
 use crate::cipher_suite::SSLCipherSuite;

--- a/security-framework/Cargo.toml
+++ b/security-framework/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "security-framework"
-version = "2.1.0"
+version = "2.1.1"
 authors = ["Steven Fackler <sfackler@gmail.com>", "Kornel <kornel@geekhood.net>"]
 license = "MIT OR Apache-2.0"
 description = "Security.framework bindings for macOS and iOS"
@@ -14,7 +14,7 @@ exclude = ["test/*"]
 edition = "2018"
 
 [dependencies]
-security-framework-sys = { version = "2.1.0", default-features = false, path = "../security-framework-sys" }
+security-framework-sys = { version = "2.1.1", default-features = false, path = "../security-framework-sys" }
 core-foundation = "0.9"
 core-foundation-sys = "0.8"
 bitflags = "1.2.1"

--- a/security-framework/src/base.rs
+++ b/security-framework/src/base.rs
@@ -1,11 +1,12 @@
 //! Support types for other modules.
 
+#[cfg(target_os = "macos")]
 use core_foundation::string::CFString;
 use core_foundation_sys::base::OSStatus;
 use std::error;
 use std::fmt;
-use std::result;
 use std::num::NonZeroI32;
+use std::result;
 
 /// A `Result` type commonly returned by functions.
 pub type Result<T, E = Error> = result::Result<T, E>;

--- a/security-framework/src/certificate.rs
+++ b/security-framework/src/certificate.rs
@@ -12,6 +12,8 @@ use std::ptr;
 
 use crate::base::{Error, Result};
 #[cfg(any(feature = "OSX_10_12", target_os = "ios"))]
+use crate::{cvt, key};
+#[cfg(any(feature = "OSX_10_12", target_os = "ios"))]
 use core_foundation::base::FromVoid;
 #[cfg(any(feature = "OSX_10_12", target_os = "ios"))]
 use core_foundation::number::CFNumber;
@@ -30,8 +32,6 @@ use security_framework_sys::trust::{
 };
 #[cfg(any(feature = "OSX_10_12", target_os = "ios"))]
 use std::ops::Deref;
-#[cfg(any(feature = "OSX_10_12", target_os = "ios"))]
-use crate::{cvt, key};
 
 declare_TCFType! {
     /// A type representing a certificate.

--- a/security-framework/src/os/macos/item.rs
+++ b/security-framework/src/os/macos/item.rs
@@ -78,11 +78,11 @@ impl ItemSearchOptionsExt for ItemSearchOptions {
 
 #[cfg(test)]
 mod test {
-    use tempdir::TempDir;
-    use crate::os::macos::item::ItemSearchOptionsExt;
     use crate::item::*;
     use crate::os::macos::certificate::SecCertificateExt;
+    use crate::os::macos::item::ItemSearchOptionsExt;
     use crate::os::macos::test::keychain;
+    use tempdir::TempDir;
 
     #[test]
     fn find_certificate() {

--- a/security-framework/src/os/macos/mod.rs
+++ b/security-framework/src/os/macos/mod.rs
@@ -17,13 +17,13 @@ pub mod transform;
 
 #[cfg(test)]
 pub mod test {
+    use crate::identity::SecIdentity;
+    use crate::item::{ItemClass, ItemSearchOptions, Reference, SearchResult};
+    use crate::os::macos::item::ItemSearchOptionsExt;
+    use crate::os::macos::keychain::SecKeychain;
     use std::fs::File;
     use std::io::prelude::*;
     use std::path::Path;
-    use crate::os::macos::item::ItemSearchOptionsExt;
-    use crate::identity::SecIdentity;
-    use crate::item::{ItemClass, ItemSearchOptions, Reference, SearchResult};
-    use crate::os::macos::keychain::SecKeychain;
 
     pub fn identity(dir: &Path) -> SecIdentity {
         // FIXME https://github.com/rust-lang/rust/issues/30018

--- a/security-framework/src/secure_transport.rs
+++ b/security-framework/src/secure_transport.rs
@@ -104,10 +104,10 @@ use crate::base::{Error, Result};
 use crate::certificate::SecCertificate;
 use crate::cipher_suite::CipherSuite;
 use crate::identity::SecIdentity;
+use crate::import_export::Pkcs12ImportOptions;
 use crate::policy::SecPolicy;
 use crate::trust::SecTrust;
 use crate::{cvt, AsInner};
-use crate::import_export::Pkcs12ImportOptions;
 use security_framework_sys::base::errSecParam;
 
 /// Specifies a side of a TLS session.

--- a/security-framework/src/trust.rs
+++ b/security-framework/src/trust.rs
@@ -12,7 +12,7 @@ use crate::certificate::SecCertificate;
 use crate::cvt;
 use crate::key::SecKey;
 use crate::policy::SecPolicy;
-use core_foundation::error::{CFErrorRef, CFError};
+use core_foundation::error::{CFError, CFErrorRef};
 
 /// The result of trust evaluation.
 #[derive(Debug, Copy, Clone, PartialEq, Eq)]

--- a/systest/src/main.rs
+++ b/systest/src/main.rs
@@ -9,6 +9,8 @@ use security_framework_sys::access::*;
 use security_framework_sys::authorization::*;
 use security_framework_sys::base::*;
 use security_framework_sys::certificate::*;
+#[cfg(target_os = "macos")]
+use security_framework_sys::certificate_oids::*;
 use security_framework_sys::cipher_suite::*;
 #[cfg(target_os = "macos")]
 use security_framework_sys::digest_transform::*;
@@ -27,8 +29,6 @@ use security_framework_sys::random::*;
 use security_framework_sys::secure_transport::*;
 #[cfg(target_os = "macos")]
 use security_framework_sys::transform::*;
-#[cfg(target_os = "macos")]
-use security_framework_sys::certificate_oids::*;
 use security_framework_sys::trust::*;
 use security_framework_sys::trust_settings::*;
 

